### PR TITLE
Fix autoDetection to accept language aliases

### PR DESCRIPTION
**Pull request by egor-rogov**
Bug introduced by #1834, found in #1868.

### CODE REVIEW

File: src/highlight.js [hunk 1]
```diff
@@ -707,8 +707,8 @@ https://highlightjs.org/
   }
 
   function autoDetection(name) {
-    name = (name || '').toLowerCase();
-    return !languages[name].disableAutodetect;
+    lang = getLanguage(name);
+    return lang && !lang.disableAutodetect;
```
Comment (marcoscaceres) [inline src/highlight.js]: Does lang have to be a global or closure? Could it be a local var?

Comment (egor-rogov) [reply src/highlight.js]: Yep, you're right of course.

**Comment by alazurenko** [comment]
Hello guys,
Do you know how soon this fix could be shipped with a new version?

**Comment by egor-rogov** [comment]
@alazurenko I guess pretty soon, as @marcoscaceres has plans to issue a minor release (for #1872).

**Comment by marcoscaceres** [comment]
Yep, will get out tomorrow. It’s a little after midnight here, so want to be sure I’m around when we release in case anything goes wrong 🙏

**Pull request merged**